src/app/footer/footer.component.scss

```diff
@@ -59,6 +60,7 @@
     &:focus,
     &:hover {
       color: $gold;
+      text-decoration: none;
     }
```

Comment:
The nesting is a little odd here, but perhaps this repo is just set to a two space tab